service/src/CpufreqSysfsDriver.cpp

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "config.h"
+
+#include "CpufreqSysfsDriver.hpp"
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "SysfsIOGroup.hpp"
+#include "geopm/Helper.hpp"
+#include "geopm/PlatformTopo.hpp"
+#include "geopm_topo.h"
+
+#include <cmath>
+#include <cstring>
+#include <sstream>
```

Comment:
style: Move up under the c headers.  Look at PlatformIO.cpp as an example.

Desired ordering:
specific class hpp, c headers, c++ headers, GEOPM public headers, other GEOPM headers

Reply:
I've updated my formatter config to sort by:

-1: config.h
0: self header or system under test's header for unit test files
1: C system headers `'<[[:alnum:]._\/]+\.h>' `
2: C++ system headers `'<[[:alnum:].]+>'`
3: Local C headers (generally our public headers. Also including 3rd party such as contrib/gtest/gmock/sqlite): `'"[[:alnum:]._\/]+\.h"'`
4: Local C++ headers (generally our non-public headers. Also including 3rd party): `'"[[:alnum:]._\/]+\.hpp"'`
5: Anything else

Seem okay? Or incorporate 3rd party as part of 1 and 2 or their own sections?

Reply:
Maybe also augment 3 to include both the .h headers and the `geopm/` .hpp headers?

Reply:
Discussed in standup:
Try for 3rd party as the first group after system headers

Yes, use .h and geopm/*.hpp in group 3